sql/core/src/main/scala/org/apache/spark/sql/internal/SharedState.scala

```diff
@@ -228,7 +211,8 @@ object SharedState extends Logging {
   def loadHiveConfFile(
       sparkConf: SparkConf,
       hadoopConf: Configuration,
-      initialConfigs: scala.collection.Map[String, String] = Map.empty): Unit = {
+      initialConfigs: scala.collection.Map[String, String] = Map.empty):
+    (SparkConf, Configuration) = {
```

Comment:
format:
```
      initialConfigs: scala.collection.Map[String, String] = Map.empty)
    : (SparkConf, Configuration) = {
```

Reply:
Or We can return  `initialConfigs -- Seq(WAREHOUSE_PATH.key, hiveWarehouseKey)` instead?